services/pg-backup/docker-compose.yml.j2

```diff
@@ -4,6 +4,9 @@ services:
     image: kartoza/pg-backup:14-3.3@sha256:4d1867707b1da879324199fb3c0d542122113bfc50cd2418764efb8f4309d6c1
     volumes:
       - /tank/${MACHINE_FQDN}:/backups
+      - ./custom-backups-cron:/build_data/backups-cron  # https://github.com/kartoza/docker-pg-backup/issues/97#issuecomment-1812504162
```

Comment:
You bind-mount volumes here, so pg-backup then has to run on the machine that holds the files. I think with our current CI/CD pipeline no machine even has those files on its local harddrive. So I am not sure if this will work (you can try it out though, maybe it works)



You could use docker configs as well I guess

Reply:
I also see that the custom-backups script stem from discussion in the issue that you linked. Maybe you want to mkae this link more prominent at a comment at the top of some files as well, since this explains why the docker service is configured exactly like this (and not a more "normal looking" way)

Reply:
Good catch. Thank you!